src/server/blocking_controller.h

```diff
@@ -15,10 +15,11 @@
 namespace dfly {
 
 class Transaction;
+class Namespace;
 
 class BlockingController {
  public:
-  explicit BlockingController(EngineShard* owner);
+  explicit BlockingController(EngineShard* owner, Namespace* ns);
```

Comment:
I assume that BlockingController is a global creature, so this binding is i guess a workaround to  make it work with default Namespace? If I am correct, please add a TODO.

Reply:
BlockingController is actually per namespace, not global